pkg/remote/cluster_client.go

```diff
@@ -27,11 +27,11 @@ var ErrAccessSecretNotFound = errors.New("access secret not found")
 func (cc *ClusterClient) GetRestConfigFromSecret(ctx context.Context, name, namespace string) (*rest.Config, error) {
 	kubeConfigSecretList := &apicorev1.SecretList{}
 	if err := cc.DefaultClient.List(ctx, kubeConfigSecretList, &client.ListOptions{
-		LabelSelector: k8slabels.SelectorFromSet(k8slabels.Set{v1beta2.KymaName: name}), Namespace: namespace,
+		LabelSelector: k8slabels.SelectorFromSet(k8slabels.Set{shared.KymaName: name}), Namespace: namespace,
 	}); err != nil {
 		return nil, fmt.Errorf("failed to list kubeconfig secrets: %w", err)
 	} else if len(kubeConfigSecretList.Items) < 1 {
-		return nil, fmt.Errorf("secret with label %s: %w", v1beta2.KymaName, ErrAccessSecretNotFound)
+		return nil, fmt.Errorf("secret with label %s: %w", shared.KymaName, ErrAccessSecretNotFound)
```

Comment:
Would be great if we can add the Kyma name to the error for better debugging
```suggestion
		return nil, fmt.Errorf("secret with label '%s:%s' not found: %w", shared.KymaName, name ErrAccessSecretNotFound)
```

Reply:
Otherwise it looks like 

```
secret with label operator.kyma-project.io/kyma-name:
      access secret not found
```

Reply:
good point, but your commit missing dot in between `name ErrAccessSecretNotFound`, fix in next commit.